features/org.wso2.carbon.appmgt.publisher.feature/src/main/resources/publisher/themes/appm/js/resource-add.js

```diff
@@ -112,6 +112,22 @@ $( document ).ready(function() {
     });
 
     /**
+     * This will collect treat as a site values and push into hidden input filed.
+     */
```

Comment:
move the common code to resource-common.js from resource-add.js and resource-edit .js